File: WebAPK/WebAPK/Models/Order.cs
using System;
using System.Collections.Generic;

namespace WebAPK.Models;

public partial class Order
{
    public int Id { get; set; }

    public string DeliveryAddress { get; set; } = null!;

    public DateTime OrderTime { get; set; }

    public DateTime DeliveryTime { get; set; }

    public string? Comment { get; set; }

    public double OrderPrice { get; set; }

    public bool IsCancelled { get; set; }

    public int UserId { get; set; }

    public virtual List<Item> Items { get; set; } = new List<Item>();

    public virtual User User { get; set; } = null!;
}

File: Frontend/react-app/src/models/order.js
export class Order{
    constructor(obj) {
        this.id = obj.id;
        this.deliveryAddress = obj.deliveryAddress;
        this.comment = obj.comment;
        this.orderTime = obj.orderTime;
        this.deliveryTime = obj.deliveryTime;
        this.isCancelled = obj.isCancelled;
        this.orderPrice = obj.orderPrice;
        this.items = obj.items.map((o) => new ItemModel(o));
      }
}